pkg/security/registry/podsecuritypolicyselfsubjectreview/rest.go

```
@@ -0,0 +1,90 @@
+func (r *REST) New() runtime.Object {
+	return &securityapi.PodSecurityPolicySelfSubjectReview{}
+}
+
+// Create registers a given new pspssr instance to r.registry.
+func (r *REST) Create(ctx kapi.Context, obj runtime.Object) (runtime.Object, error) {
+	pspssr, ok := obj.(*securityapi.PodSecurityPolicySelfSubjectReview)
+	if !ok {
+		return nil, kapierrors.NewBadRequest(fmt.Sprintf("not a PodSecurityPolicySelfSubjectReview: %#v", obj))
+	}
+	if errs := securityvalidation.ValidatePodSecurityPolicySelfSubjectReview(pspssr); len(errs) > 0 {
+		return nil, kapierrors.NewInvalid(securityapi.Kind(pspssr.Kind), "", errs)
+	}
+	userInfo, ok := kapi.UserFrom(ctx)
+	if !ok {
+		return nil, kapierrors.NewBadRequest(fmt.Sprintf("no user data associated with context"))
+	}
+	ns, ok := kapi.NamespaceFrom(ctx)
+	if !ok {
+		return nil, kapierrors.NewBadRequest("namespace parameter required.")
+	}
+
+	matchedConstraints, err := r.sccMatcher.FindApplicableSCCs(userInfo)
+	if err != nil {
+		return nil, kapierrors.NewBadRequest(fmt.Sprintf("unable to find SecurityContextConstraints: %v", err))
+	}
+	saName := pspssr.Spec.Template.Spec.ServiceAccountName
+	if len(saName) > 0 {
+		saUserInfo := serviceaccount.UserInfo(ns, saName, "")
+		saConstraints, err := r.sccMatcher.FindApplicableSCCs(saUserInfo)
+		if err != nil {
+			return nil, kapierrors.NewBadRequest(fmt.Sprintf("unable to find SecurityContextConstraints: %v", err))
+		}
+		matchedConstraints = append(matchedConstraints, saConstraints...)
+	}
+	oscc.DeduplicateSecurityContextConstraints(matchedConstraints)
+	sort.Sort(oscc.ByPriority(matchedConstraints))
+	var namespace *kapi.Namespace
+	for _, constraint := range matchedConstraints {
+		var (
+			provider kscc.SecurityContextConstraintsProvider
+			err      error
+		)
+		if provider, namespace, err = oscc.CreateProviderFromConstraint(ns, namespace, constraint, r.client); err != nil {
+			glog.Errorf("Unable to create provider for constraint: %v", err)
```

Why you're swallowing the error here?